src/styles/index.scss

```diff
@@ -1,3 +1,222 @@
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    list-style: none;
+    display: flex;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    color: #060b35;
+    text-transform: uppercase;
+    padding-top: 22px;
+    padding-bottom: 22px;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    line-height: 16px;
+    text-decoration: none;
+    display: inline-block;
+    position: relative;
+  }
+
+  &__link--is-active {
+    color: #00acdc;
+    position: relative;
+  }
+
+  &__link--is-active::after {
+    content: ' ';
+    position: absolute;
+    bottom: 0;
+    display: block;
+    border-radius: 2px;
+    width: 100%;
+    height: 4px;
+    left: 0;
+    background: #00acdc;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
+}
+
+main {
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  display: grid;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__icon {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__info {
+    text-align: left;
+    color: #060b35;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__info:hover {
+    color: #34568b;
+  }
+
+  &__code {
+    color: #616070;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rate {
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    display: flex;
+  }
+
+  &__rate-text {
+    color: #060b35;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+    margin-bottom: 16px;
+    display: flex;
+  }
+
+  &__price-text {
+    color: #616070;
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-money {
+    color: #060b35;
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    color: #fff;
+    text-align: center;
+    background: #00acdc;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-decoration: none;
+    display: block;
+  }
+
+  &__button:hover {
+    color: #00acdc;
+    background: #fff;
+    border: 1px solid #00acdc;
+  }
+}
+
+.stars {
+  box-sizing: border-box;
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    background-position: 50%;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    background-image: url(../images/star-active.svg);
```

Comment:
Verify that the `background-image` paths are correct. The paths should be relative to the location of the compiled CSS file, not the SCSS file.